cluster/log-dump/log-dump.sh

```diff
@@ -675,6 +689,9 @@ function main() {
   fi
 
   detect_node_failures
+  if [[ ${log_dump_expected_success_percentage} -gt 0 ]]; then
```

Comment:
after switch from error code to flag indicating failure:
```
if log exporting failed; then
  if [[ ${log_dump_expected_success_percentage} -gt 0 ]]; then
     return 1
   fi
fi
```

Reply:
or combine both in one condition if more readable

Reply:
Combined both conditions into a single one. Done.

Reply:
Combined both conditions into a single one. Done.